views/common/base.njk

```diff
@@ -389,11 +389,11 @@
             <ul class="app-vertical-nav__list">
 
               <li class="app-vertical-nav__item  ">
-                <a class="app-vertical-nav__link" href="/design-system-benefits/">Benefits of using the Design System</a>
+                <a class="app-vertical-nav__link" href="/design-system-statuses/">Design system statuses</a>
               </li>
 
               <li class="app-vertical-nav__item  ">
-                <a class="app-vertical-nav__link" href="/design-system-statuses/">Design system statuses</a>
+                <a class="app-vertical-nav__link" href="/design-system-benefits/">Benefits of using the Design System</a>
```

Comment:
Are these supposed to switch over? I have no problem with it, just want to check it's been requested/seen/approved by Helen or Murray